frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/ReverificationResultService.java

```diff
@@ -71,15 +70,9 @@ public TokenRequest constructTokenRequest(String authCode) {
                         NowHelper.now(),
                         NowHelper.now(),
                         new JWTID());
-        return new TokenRequest(
-                ipvTokenURI,
-                generatePrivateKeyJwt(claimsSet),
-                codeGrant,
-                null,
-                null,
-                Map.of(
-                        "client_id",
-                        singletonList(configurationService.getIPVAuthorisationClientId())));
```

Comment:
guess removing the singletonList is fine?

Reply:
I believe so, the method `customParameter` is 

```
        public Builder customParameter(String name, String... values) {
            if (values != null && values.length != 0) {
                this.customParams.put(name, Arrays.asList(values));
            } else {
                this.customParams.remove(name);
            }

            return this;
        }
```